mwdb/web/src/types/api.ts

```diff
@@ -317,3 +327,25 @@ export type OauthRemoveSingleProviderResponse = Response<null>;
 export type OauthGetLogoutLinkResponse = Response<{
     url: string;
 }>;
+
+export type UploadConfigRequest = {
+    cfg?: string;
+    family: string;
+    parent?: string;
+    config_type?: string;
+    attributes?: Attribute[];
```

Comment:
missing `shareWith` and `share3rdParty`. I think it would be nice to split the common fields from specialized fields in that scheme (if possible)